flow-typed/npm/react-native-document-picker_vx.x.x.js

```diff
@@ -0,0 +1,33 @@
+// flow-typed signature: e70237f3f0f0f7216ac66fa336d6aecf
+// flow-typed version: <<STUB>>/react-native-document-picker_vgit://github.com/Elyx0/react-native-document-picker.git#v3/flow_v0.92.0
```

Comment:
flow types: Add type for react-native-document-picker.
    
    This dependency was added in d9bbb5e5f047a6de17eef1870fbad17c8debf5f9
    but flow type were not added.
    
    So now just run `flow-typed update` and it is generated automatically.

Heh, we actually do have types for this! Look in the parent directory, and then at the history :slightly_smiling_face: 

Or most practically: go to `ComposeMenu.js` where we use this library, in the `handleFilePicker` function, and try editing `response.name` or something to misspell the property's name. You'll get a Flow error.

I did leave things a little messy in that `yarn update-libdefs` creates this file -- really we want to leave things in a state where it makes no changes unless there are updates to be had. So thanks for tidying that up!

I'll take a few minutes to see if I can teach `yarn update-libdefs` to ignore this library, and if not then I'll merge this commit.

Reply:
Merged, as 6219e5a23 . Thanks again!